py3dtilers/TilesetReader/parsed_b3dm.py

```diff
@@ -0,0 +1,130 @@
+import os
+
+from py3dtiles import TriangleSoup, GlTFMaterial
+from ..Common import ObjectsToTile, ObjectToTile
+from ..Texture import Texture
+
+
+class ParsedB3dm(ObjectToTile):
```

Comment:
Again (and sorry for that) the semantics carried by this class name is not very helpful.
Maybe `FromTileToObjetToTile` would be clearer ? (although it is still awkward: anything better ?)